src/main/java/seedu/address/logic/commands/SampleCommand.java

```diff
@@ -20,8 +20,8 @@ public class SampleCommand extends Command {
     public static final String COMMAND_WORD = "sample";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Populate the EduMate with "
-            + "sample data of the given size.\n"
-            + "Parameters: SIZE (non-negative integer)\n"
+            + "sample data of the given size between 1 and 100.\n"
+            + "Parameters: SIZE (non-negative integer less than 100)\n"
```

Comment:
these 2 sentences contradict each other,

Reply:
Just  changed to 101.